cspyce/__init__.py

```diff
@@ -13,18 +13,30 @@
 # See the file AAREADME.txt for more information.
 #
 # Mark Showalter, PDS Ring-Moons Systems Node, SETI Institute, December 2017.
+#   - Adapted for Python 3, February 2022.
 ################################################################################
 
+import inspect
+import sys
+import os
+
+PYTHON2 = sys.version_info[0] < 3
+
 # We allow the import of cspyce2 to fail because, during development, there are
 # times when cspyce2.py might be invalid. If that happens, we still want to be
 # able to work inside the cspyce directory tree. Without allowing this
 # exception, it could become impossible to work on and test cspyce functions
 # inside this directory tree. This should never occur during a normal run.
 
 try:
-    from cspyce2 import *
-except ImportError:
-    pass
+    from .cspyce2 import *
+except ImportError as err:
+    if os.getenv("CSPICE_DEVELOPMENT"):
+        print("Package cspyce2 not found.  Error ignored.")
+        pass
+    else:
+        print("Set environment variable 'CSPICE_DEVELOPMENT' to 'True' to ignore this error")
```

Comment:
I don't think you can set the environment variable to "True". You have to set it to "1", right?

Reply:
Anything that isn't the empty string will do.  I fixed this for you.